### PR TITLE
Add precision badge to localized money input

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

Add precision badge to localized money input.

### CODE REVIEW

**Comment by FilPob** [comment]
Looks good from my side. Thanks :)

**Review by kark — APPROVED** [review]
Thanks 💯

File: .changeset/rude-radios-smell.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/localized-money-input': minor
+---
+
+add precision badge to localized money input field
```
Comment (CarlosCortizasCT) [inline .changeset/rude-radios-smell.md:5]:
```suggestion
Added a new property (**hasHighPrecisionBadge**) which can be used so the component shows a high precision badge when its value has a high precision decimal number.
```

File: packages/components/inputs/localized-money-input/src/localized-money-input.visualroute.jsx [hunk 2]
```diff
@@ -140,5 +147,23 @@ export const component = () => (
...
+    </Spec>
+    <Spec label="with high precision badge and high precision price">
+      <LocalizedMoneyInput
+        value={highPrecisionValue}
+        onChange={() => {}}
+        selectedCurrency="CAD"
+        horizontalConstraint={7}
+        hasHighPrecisionBadge={true}
+      />
+    </Spec>
```
Comment (CarlosCortizasCT) [inline packages/components/inputs/localized-money-input/src/localized-money-input.visualroute.jsx:167]: 💯

**Review by CarlosCortizasCT — APPROVED** [review]
I left a suggestion to adjust the changeset message, but otherwise looks good to me.

Thanks 👍

File: .changeset/rude-radios-smell.md (hunk 1 above)
Comment (ddouglasz) [reply .changeset/rude-radios-smell.md]: Thank you for the suggestion, updated here: https://github.com/commercetools/ui-kit/pull/2578/commits/e87ed73a1e7db4bb95602168669a7572b20d4bde

**Pull request merged**